ci-operator/step-registry/workers-rhel/pre-hook/sshtunnel/workers-rhel-pre-hook-sshtunnel-commands.sh

```diff
@@ -0,0 +1,205 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+set -x
+
+
+export QE_SSH_PRIVATE_KEY=/var/run/qe/private_key
```

Comment:
How about `SSH_PRIV_KEY_PATH=${CLUSTER_PROFILE_DIR}/ssh-privatekey` ?

Reply:
yes, it is from dev as we use same cluster-profile on vsphere. If we inject dev ssh public key into bastion host, here `${CLUSTER_PROFILE_DIR}/ssh-privatekey` can be used directly, wdyt? Currently, rhel8 image already inject dev ssh public key for ansible usage.

Reply:
Yeah, that is what I am thinking, maybe we can inject both openshift-qe and dev's public key in our bastion host, so that we do not have to mount QE dedicated vault anywhere.

Reply:
ok, will do that.